Cargo.toml

```diff
@@ -11,7 +11,7 @@ rust-version = "1.86"
 [dependencies]
 anylog = "0.6.3"
 anyhow = { version = "1.0.69", features = ["backtrace"] }
-apple-catalog-parsing = { path = "apple-catalog-parsing", optional = true }
```

Comment:
@szokeasaurusrex, I had to remove the optional flag to get this to compile properly. Admittedly I'm a bit unsure of the effect that removing this optionality would have as it pertains to the release (and how you had the dependency seemingly conditionally included with the `unstable-mobile-app` feature), so if you don't mind a check/fix here, that would be great.

Reply:
I think this is fine. The crate should compile on all platforms; on non-macOS platforms it is just an empty crate.

I can try to see though if I can add it as a macOS-specific dependency